config.json

```diff
@@ -249,7 +249,10 @@
       "unlocked_by": null,
       "difficulty": 1,
       "topics": [
-
+        "loops",
+        "parsing",
+        "transforming",
+        "text formatting"
```

Comment:
We've switched to `snake_case` naming convention for topics to increase consistency among tracks, so could you please choose them from the following list - https://github.com/exercism/problem-specifications/blob/master/TOPICS.txt